pymmcore_plus/_tests/conftest.py

```diff
@@ -1,17 +1,21 @@
 import pytest
 
 import pymmcore_plus
+from pymmcore_plus.core.events import CMMCoreSignaler, QCoreSignaler
+from pymmcore_plus.mda.events import MDASignaler, QMDASignaler
 
 
 @pytest.fixture(params=["QSignal", "psygnal"], scope="function")
 def core(request):
     core = pymmcore_plus.CMMCorePlus()
     if request.param == "psygnal":
-        core.events = pymmcore_plus.CMMCoreSignaler()
-        core._callback_relay = pymmcore_plus.core._mmcore_plus.MMCallbackRelay(
-            core.events
-        )
-        core.registerCallback(core._callback_relay)
+        core.events = CMMCoreSignaler()
+        core.mda._events = MDASignaler()
+    else:
+        core.events = QCoreSignaler()
+        core.mda._events = QMDASignaler()
+    core._callback_relay = pymmcore_plus.core._mmcore_plus.MMCallbackRelay(core.events)
```

Comment:
also of note - I don't think we were garunteed to get the QSignaler before. At least locally I wasn't getting it.